demos/wordcount/src/main/java/com/datatorrent/demos/wordcount/ApplicationWithQuerySupport.java

```diff
@@ -38,13 +38,33 @@
 
 import org.apache.hadoop.conf.Configuration;
 
+/**
+ * Simple demo that counts word frequencies from any file dropped into a
+ * monitored directory. It outputs the top N word-frequency pairs for each file
+ * as well globally across all files.
+ * <p>
+ * Each input file generates a corresponding output file in the output directory
+ * containing the top N pairs for that file. The output is also written
+ * to an internal store to support visualization in the UI via queries.
+ * <p>
+ * @since 3.2.0
```

Comment:
In this case it is required since due to issues with the javadoc it could not be added as part of the release.